modules/programs/firefox/mkFirefoxModule.nix

```diff
@@ -801,7 +812,28 @@ in {
       }
 
       (mkNoDuplicateAssertion cfg.profiles "profile")
-    ] ++ (mapAttrsToList
+    ] ++ (
+
+      # Assert "toolkit.legacyUserProfileCustomizations.stylesheets" is enabled if userChrome/userContent is used.
+      let
+
+        assertProfile = userChromeAttr: profile:
+          profile.${userChromeAttr} != ""
+          -> (profile.settings."toolkit.legacyUserProfileCustomizations.stylesheets"
+            != false);
+
+        mkAssertion = userChromeAttr: {
+          assertion =
+            all (assertProfile userChromeAttr) (attrValues cfg.profiles);
+          message = ''
+            {option}`${userChromeAttr}` won't work with
+            {option}`settings."toolkit.legacyUserProfileCustomizations.stylesheets"` set to false.
```

Comment:
I'm not sure if you saw my other comment on this since it was in a resolved thread, but `` {option}`x` `` shouldn't be used in assertions, because they don't get rendered like the docs do.

Reply:
Ok. I left the backticks characters.